recipes/pyld/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "PyLD" %}
+{% set version = "0.7.3" %}
+{% set sha256 = "4baafb5487e4c2105479f15cac5afb1d0f6606d18eddc1da1504bb2267c7a602" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/da/c7/1a87baf08fca47cb17657ba2057c7f10eadaff38e39cf0210f1bd012d623/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install
+
+requirements:
+  build:
+    - python
+    - filelock
```

Comment:
I don't see `filelock` mentioned anywhere in the `setup.py` or the project. Is it actually required here and in the `run` section.